### ISSUE TITLE
Error: Required named parameter 'lineInfo' must be provided.

### ISSUE THREAD

**Issue by liudonghua123** [maks/dart_repl#8]
I tried to install the latest 0.1.0 of this package. Then I got the following errors.

```shell
D:\code\other>dart pub global activate repl
Resolving dependencies...
+ _fe_analyzer_shared 38.0.0 (40.0.0 available)
+ analyzer 3.4.1 (4.1.0 available)
+ async 2.9.0
+ cli_repl 0.2.2
+ collection 1.16.0
+ convert 3.0.1
+ crypto 3.0.2
+ file 6.1.2
+ glob 2.0.2
+ js 0.6.4
+ meta 1.8.0
+ package_config 2.0.2
+ path 1.8.2
+ pub_semver 2.1.1
+ repl 0.1.0
+ source_span 1.9.0
+ string_scanner 1.1.1
+ term_glyph 1.2.0
+ typed_data 1.3.1
+ vm_service 8.3.0
+ watcher 1.0.1
+ yaml 3.1.1
Downloading repl 0.1.0...
Downloading cli_repl 0.2.2...
Downloading vm_service 8.3.0...
Downloading meta 1.8.0...
Downloading string_scanner 1.1.1...
Downloading path 1.8.2...
Downloading source_span 1.9.0...
Downloading async 2.9.0...
Building package executables...
Built repl:scratchpad.
Failed to build repl:main:
/C:/Users/Liu.D.H/AppData/Local/Pub/Cache/hosted/pub.flutter-io.cn/repl-0.1.0/lib/parser.dart:35:13: Error: Required named parameter 'lineInfo' must be provided.
      Parser(StringSource(code, ''), errorListener, featureSet: featureSet);
            ^
/C:/Users/Liu.D.H/AppData/Local/Pub/Cache/hosted/pub.flutter-io.cn/analyzer-3.4.1/lib/src/generated/parser.dart:32:3: Context: Found this candidate, but the arguments don't match.
  Parser(Source source, AnalysisErrorListener errorListener,
  ^^^^^^

D:\code\other>flutter --version && dart --version
Flutter 3.0.1 • channel stable • git@github.com:flutter/flutter.git
Framework • revision fb57da5f94 (3 days ago) • 2022-05-19 15:50:29 -0700
Engine • revision caaafc5604
Tools • Dart 2.17.1 • DevTools 2.12.2
Dart SDK version: 2.17.1 (stable) (Tue May 17 17:58:21 2022 +0000) on "windows_x64"

D:\code\other>ver

Microsoft Windows [Version 10.0.22000.675]

D:\code\other>
```

**Comment by liudonghua123** [comment]
I make a pr https://github.com/maks/dart_repl/pull/9 for this.

**Comment by maks** [comment]
Thank you for reporting this and submitting a fix!
Unfortunately this was the result of my rushing to get the release out and not getting around to setting up some basic CI, which I've now done.

**Comment by maks** [comment]
Fixed in #9
